training/finetune_acft_hf_nlp/src/finetune/finetune.py

```diff
@@ -435,6 +444,44 @@ def finetune(args: Namespace):
     hf_task_runner = get_task_runner(task_name=args.task_name)()
     hf_task_runner.run_finetune(args)
 
+    process_name = os.environ.get('AZUREML_PROCESS_NAME', 'main')
+    logger.info(f"Process - {process_name}")
+    if process_name in {'main', 'rank_0'}:
+        # hotfix mlflow model deployment issue
+        mlflow_model_path = args.mlflow_model_folder
```

Comment:
this fix needs to be only for `mlflow_model_model` is not empty right?

Reply:
if the files do not exist the try block will catch them

Reply:
lines 452 and 453 will fail if mlflow_model_path is None

Reply:
yes, moved inside try block